dev/com.ibm.ws.webcontainer.servlet.3.1_fat/test-applications/TestServlet31.war/resources/plus+sign.jsp

```diff
@@ -0,0 +1,12 @@
+<%@ page language="java" contentType="text/html; charset=ISO-8859-1"
+    pageEncoding="ISO-8859-1"%>
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="ISO-8859-1">
+<title>Insert title here</title>
+</head>
+<body>
+This file has a plus sign in the name.
+</body>
+</html>
```

Comment:
copyright comment here as well

Reply:
This file is replaced by an html file, which has a copyright.